nltk/classify/senna.py

```diff
@@ -30,9 +30,9 @@
 Note: Unit tests for this module can be found in test/unit/test_senna.py
 
 >>> from nltk.classify import Senna
->>> pipeline = Senna('/usr/share/senna-v3.0', ['pos', 'chk', 'ner'])
+>>> pipeline = Senna('/usr/share/senna-v3.0', ['pos', 'chk', 'ner'])  # doctest: +SKIP
```

Comment:
All `Senna(...)` sections are skipped, as they break on the Windows and Mac CI's, because the `SENNA` environment variable [does exist](https://github.com/nltk/nltk/blob/develop/.github/workflows/ci.yaml#L11), but points to a path that doesn't.